app/src/main/res/layout/settings.xml

```diff
@@ -48,4 +48,28 @@
         android:layout_height="50dp"
         android:checked="true"/>
 
-</LinearLayout>
\ No newline at end of file
+    <Switch
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:text="Auto Broadcast"
+        android:id="@+id/autoBroadcast"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:checked="true"/>
+
+    <EditText
+        android:id="@+id/autoBroadcastText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="10dp"
+        android:layout_marginLeft="10dp"
+        android:layout_marginTop="10dp"
+        android:ems="10"
+        android:hint="Enter broadcast interval in miliseconds"
```

Comment:
You probably want this to be seconds.  You also want some guard rails on the value to prevent the user from specifying a value too low (i.e.: Less than the average time of a COT update plus some margin). I would imagine putting a "1" as a value would not behave well.  Maybe 5s - 100,000s?  That gives you an interval range of well over 24 hours

Reply:
Yeah the EditText is not the best choice here, maybe a NumberPicker that has set values like 1 minute, 10 minutes, 60 minutes, 120 minutes, ... etc